bonfire/namespaces.py

```diff
@@ -283,6 +283,9 @@ def reserve_namespace(name, requester, duration, pool, timeout, local=True):
         pool,
     )
 
+    # set reserved namespace as current
+    set_current_namespace(ns_name)
```

Comment:
Let's also disable this behavior when in 'bot' mode:

```suggestion
    if not conf.BONFIRE_BOT:
        set_current_namespace(ns_name)
```

Reply:
Makes sense. It would be better to not use any specialties like `BONFIRE_BOT` but rather have it universal and have an option for almost each action that needs to be configurable.